content/docs/configuration-system/content.md

```diff
@@ -0,0 +1,220 @@
++++
+fragment = "content"
+weight = 100
+title = "Configuration System"
+subtitle = "knoxite's configuration system"
+
+[sidebar]
+  sticky = true
++++
+
+<p>
+
+Knoxite provides a configuration system for your repositories. You can declare
+shorthands to your repositories and provide default values for settings like
+encryption, compression or excludes.
+The `config` command offers a variety of subcommands in order help you to manage
+your configuration.
+
+A configuration is structured in [json](https://json.org) format and does look
+like this:
+```json
+{
+    "repositories": {
+        "fort": {
+            "url": "/tmp/knox",
+            "compression": "",
+            "tolerance": 0,
+            "encryption": "",
+            "store_excludes": null,
+            "restore_excludes": null
+        }
+    }
+}
+```
+
+### Initializing the configuration
+
+Before we can start composing our configuration we initialize the empty file
```

Comment:
nitpick: **we need to initialize**, otherwise the "before, first constellation" doesn't work.